src/ipfs.ts

```diff
@@ -0,0 +1,53 @@
+import { Readable } from 'stream';
+
+import { CID } from 'ipfs-http-client';
+
+import { toString } from './merkle.js';
+import { shared as Shared } from './shared.js';
+import { MerkleTree } from './types.js';
+
+// TODO: local IPFS node required for the development and tests
+export async function uploadTree(t: MerkleTree): Promise<CID> {
+    const data = toString(t);
+
+    return Shared.IPFS.pinFileToIPFS(Readable.from(data), {
+        pinataMetadata: {
+            name: 'eth2-fees',
+        },
+    });
+}
+
+export async function catCID(cid: string): Promise<string> {
+    const r = await fetch(`https://ipfs.io/ipfs/${cid}`);
+    return await r.text();
+    return `format: standard-v1
```

Comment:
unreachable code

Reply:
It's instead of comment :) I will remove it eventually.